src/sentry/static/sentry/app/views/settings/organization/navigationConfiguration.tsx

```diff
@@ -20,6 +20,16 @@ const organizationNavigation: NavigationSection[] = [
         description: t("View and manage an organization's projects"),
         id: 'projects',
       },
+      {
+        path: `${pathPrefix}/security-and-privacy/`,
+        title: t('Security & Privacy'),
+        description: t(
+          'View and manage the security and privacy settings of an organization'
+        ),
+        id: 'security-and-privacy',
+        show: ({features}) => features!.has('datascrubbers-v2'),
```

Comment:
Does this mean that people without `datascrubbers-v2` will not see the old security settings at all?